Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -668,4 +668,32 @@ def test_search
   ensure
     desc_cache.unlink
   end
+
+  def test_link_unlink
+    assert_match "This command requires a keg argument", cmd_fail("link")
+
+    formula_file = setup_test_formula "testball"
+    cmd("install", "testball")
+    assert_match "Already linked: #{HOMEBREW_CELLAR}/testball/0.1",
```

Comment:
As we're not testing that `install` works or doesn't in this function you can skip the `assert_match` here. `cmd` is sufficient as it'll verify the return code.